tests/network-tests/src/fixtures/council/ElectCouncilFixture.ts

```diff
@@ -5,16 +5,32 @@ import { blake2AsHex } from '@polkadot/util-crypto'
 import { GenericAccountId } from '@polkadot/types'
 import { assert } from 'chai'
 import { createType, registry } from '@joystream/types'
+import { QueryNodeApi } from '../../QueryNodeApi'
+import { Api } from '../../Api'
 
 export class ElectCouncilFixture extends BaseQueryNodeFixture {
+  protected candidatesAddresses: string[]
+
+  public constructor(api: Api, query: QueryNodeApi, candidatesAddresses: string[] = []) {
+    super(api, query)
+    this.candidatesAddresses = candidatesAddresses
+  }
+
   public async execute(): Promise<void> {
     const { api, query } = this
-    const { councilSize, minNumberOfExtraCandidates } = api.consts.council
-    const numberOfCandidates = councilSize.add(minNumberOfExtraCandidates).toNumber()
+    const councilSize = this.api.consts.council.councilSize
+    const numberOfCandidates = this.candidatesAddresses.length
     const numberOfVoters = numberOfCandidates
 
-    // Prepare memberships
-    const candidatesMemberAccounts = (await this.api.createKeyPairs(numberOfCandidates)).map(({ key }) => key.address)
+    // get/create candidates member accounts
+    const candidatesMemberAccounts =
+      this.candidatesAddresses.length === 0
+        ? (await this.api.createKeyPairs(numberOfCandidates)).map(({ key }) => key.address)
```

Comment:
If `this.candidatesAddresses.length` then `numberOfCandidates` is also `0`, I think you need to get `numberOfCandidates` from `api.consts` again.